test/execute_build_test.go

```diff
@@ -82,10 +81,14 @@ func testCreateImage(t *testing.T, when spec.G, it spec.S) {
 
 		deleteNamespace(t, ctx, clients, testNamespace)
 
+		labels := readNamespaceLabelsFromEnv()
```

Comment:
Is there a reason this should be a specific config option?

Reply:
as opposed to just reading the env when we need it? Not really and I can see why it feels out of place compared to the other config variables